posthog/session_recordings/session_recording_api.py

```diff
@@ -1506,22 +1506,11 @@ def list_recordings_from_query(
     if (all_session_ids and query.session_ids) or not all_session_ids:
         modifiers = safely_read_modifiers_overrides(str(user.distinct_id), team) if user else None
 
-        use_multiple_sub_queries = (
-            posthoganalytics.feature_enabled(
-                "use-multiple-sub-queries",
-                str(user.distinct_id),
-            )
-            if user
-            else False
-        )
```

Comment:
i missed that this was unused, we're only using it to tag the span, not for any "real"reason